File: JavaScript/1064 - Positivos e Média.js
var valores = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').map(Number);
var positivos = valores.filter(v => v > 0);
console.log(`${positivos.length} valores positivos`);
console.log((positivos.reduce((a,b) => a+b, 0) / positivos.length).toFixed(1));

File: Clojure/1064 - Positivos e Média.clj
(defn cont-pos [x]
    (count (filter pos? x))
    )

(defn media [y]
    (/ (reduce + (filter pos? y)) (cont-pos y))
    )

(defn main []
    (def valores (repeatedly 6 read))
    (println (cont-pos valores) "valores positivos")
    (printf "%.1f\n" (media valores))
    )

(main)